app/models/learning_hour.rb

```diff
@@ -52,7 +52,6 @@ def user_org_learning_topic_enable?
 #  id                     :bigint           not null, primary key
 #  duration_hours         :integer          not null
 #  duration_minutes       :integer          not null
-#  learning_type          :integer          default(5)
```

Comment:
interesting diff - at some point we need to figure out why this keeps appearing and disappearing

Reply:
Agreed. This thing drives me nuts in my code. Also I feel like I messed it up. @elasticspoon any ideas?

Reply:
I think this should be removed. It looks like `:learning_type` was adding in https://github.com/rubyforgood/casa/blob/bb074c4076a858e45abf493046454e3c0c25f6fd/db/migrate/20220324141758_create_learning_hours.rb#L5 but then removed in https://github.com/rubyforgood/casa/blob/bb074c4076a858e45abf493046454e3c0c25f6fd/db/migrate/20230730103110_remove_learning_type.rb#L1
It probably should have been removed but somehow got left in?

I suspect what is going on is it exists in the schema because someone forgot to commit the removal. Most people do a `schema:load` to they have it. But when people run migrations it attempts to remove the column. And every time the removal is attempted we think "thats weird, we should leave the column".

Reply:
I would leave it in for now simply because I think the removal might mess up the seeding.

Reply:
@nepaakash revert this change for now